src/prefect/settings.py

```diff
@@ -1391,16 +1391,6 @@ def default_cloud_ui_url(settings, value):
 Whether or not to enable flow run states on the flow run graph.
 """
 
-PREFECT_EXPERIMENTAL_ENABLE_WORK_POOLS = Setting(bool, default=True)
```

Comment:
When we remove these experimental settings we also want to add them to the set of `REMOVED_EXPERIMENTAL_FLAGS` in this same file: https://github.com/PrefectHQ/prefect/blob/692642b545525a3617f83708b7bc976b23a3b68e/src/prefect/settings.py#L107-L114

The reason for this is removing the settings entirely causes the CLI to crash for anyone who has them in one or more of their profiles. Adding them to `REMOVED_EXPERIMENTAL_FLAGS` will make it so that the user is warned about it and they have time to take action.

Reply:
Sure. Makes sense!